lsp-pwsh.el

```diff
@@ -266,7 +297,12 @@ Must not nil.")
   :server-id 'pwsh-ls
   :priority 1
   :initialization-options #'lsp-pwsh--extra-init-params
-  :notification-handlers (lsp-ht ("powerShell/executionStatusChanged" 'ignore))
+  :notification-handlers (lsp-ht ("powerShell/executionStatusChanged" #'ignore)
```

Comment:
not part of this review but do these have more meaningful handling?

Reply:
This's only make sense when we want to have integrated powershell console when open powershell file, like in VsCode. For just code intelligent, it's nearly useless.